SPECS/dracut/20overlayfs/overlayfs-mount.sh

```diff
@@ -51,6 +51,11 @@ parse_kernel_cmdline_args() {
 
     # Retrieve the OverlayFS parameters.
     [ -z "${overlayfs}" ] && overlayfs=$(getarg rd.overlayfs=)
+
+    # Retrieve the optional OverlayFS mount options from the kernel cmdline.
+    overlayfs_mnt_option=$(getarg rd.overlayfs_mnt_option=)
+    # If the argument is not provided, fallback to default "ro".
+    [ -z "${overlayfs_mnt_option}" ] && overlayfs_mnt_option="rw,nodev,nosuid,nouser,noexec"
```

Comment:
Why does this apply to all the overlays? Shouldn't this be specified for each overlay?